stubs/NormalizerInterface.stub

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+interface NormalizerInterface
+{
+    /**
+     * @param mixed $object
```

Comment:
could be `object`, right?

Reply:
it could be... but could be a stream, resource, scalar or other thing, the supports method limits what it can be... 

but i think most times will be object.

should i change to `mixed|object`?

Reply:
just to be sure, symfony's serializer does allow for non-objects to be "normalized", i never used it, but is a option.

https://github.com/symfony/serializer/blob/5.x/Serializer.php#L236

Reply:
`mixed|object` would be the same as `mixed`.

Reply:
fixed.